stable/superset/templates/deployment-worker.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "superset.fullname" . }}-worker
+  labels:
+    {{- include "superset.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.superset.worker.autoscaling.enabled}}
+  replicas: {{ .Values.superset.worker.replicas }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "superset.selectorLabels" . | nindent 6 }}
```

Comment:
You need different selector labels here as this is the same as the main deployment